playground/frontend/playground_components/assets/translations/en.yaml

```
@@ -26,6 +26,7 @@ errors:
   loading: 'Error while loading.'
   loadingCatalog: 'Cannot load the example catalog.'
   loadingExample: 'Cannot load the example.'
+  failedLoadExampleWithToken: 'Failed to load the example with the token: {token}.'
```

There is no concept of tokens to the end-user.
```suggestion
  failedLoadExampleWithToken: 'Failed to load the example: {token}.'
```